src/vs/gitpod/node/gitpodInsightsAppender.ts

```diff
@@ -214,6 +214,66 @@ function mapTelemetryData(eventName: string, data: any): RemoteTrackMessage | un
 					timestamp: data.timestamp
 				},
 			};
+		case 'settingsEditor.settingModified':
+			return {
+				event: 'vscode_update_configuration',
+				properties: {
+					key: data.key,
+					target: data.target,
+					workspaceId: data.workspaceId,
+					workspaceInstanceId: data.workspaceInstanceId,
+					sessionID: data.sessionID,
+					timestamp: data.timestamp
+				},
+			};
+		case 'extensionGallery:install':
+			return {
+				event: 'vscode_extension_gallery',
+				properties: {
+					kind: 'install',
+					id: data.id,
+					workspaceId: data.workspaceId,
+					workspaceInstanceId: data.workspaceInstanceId,
+					sessionID: data.sessionID,
+					timestamp: data.timestamp
+				},
+			};
+		case 'extensionGallery:update':
+			return {
+				event: 'vscode_extension_gallery',
+				properties: {
+					kind: 'update',
+					id: data.id,
+					workspaceId: data.workspaceId,
+					workspaceInstanceId: data.workspaceInstanceId,
+					sessionID: data.sessionID,
+					timestamp: data.timestamp
+				},
+			};
+		case 'extensionGallery:uninstall':
+			return {
+				event: 'vscode_extension_gallery',
+				properties: {
+					kind: 'uninstall',
+					id: data.id,
+					workspaceId: data.workspaceId,
+					workspaceInstanceId: data.workspaceInstanceId,
+					sessionID: data.sessionID,
+					timestamp: data.timestamp
+				},
+			};
+		case 'galleryService:query':
+			return {
+				event: 'vscode_extension_gallery_query',
```

Comment:
can be `vscode_extension_gallery` as well?

i.e. we can have a look at `vscode_extension_gallery` to learn whether a user try to install or manage extensions

Reply:
you mean change `vscode_extension_gallery_query` -> `vscode_extension_gallery` right? updated the event name